readme.md

```diff
@@ -479,6 +479,7 @@
 ## Decentralized Systems
 
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin) - Bitcoin services and tools for software developers.
+- [Bitcoin Cash](https://github.com/dsmurrell/awesome-bitcoin-cash) - Bitcoin Cash resources.
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**